src/connector.js

```diff
@@ -356,46 +368,40 @@ function initTpl() {
 	if ( !suggestionsElement && searchBoxElement && params.unsupportedSuggestions && params.numberOfSuggestions > 0 ) {
 		suggestionsElement = document.createElement( "ul" );
 		suggestionsElement.id = "suggestions";
+		suggestionsElement.role = "listbox";
 		suggestionsElement.classList.add( "rough-experimental", "query-suggestions" );
 
 		searchBoxElement.after( suggestionsElement );
+		searchBoxElement.setAttribute('aria-controls', 'suggestions');
 	}
 
-	// Query suggestions
-	if ( suggestionsElement ) {
+	// Rough Experiement
+	if ( roughExperiment ) {
```

Comment:
This entire `if` can be removed; we don't need to add the alert anymore. Except for the event listeners you have added in.